decidim-admin/app/commands/decidim/admin/bulk_action.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class BulkAction < Decidim::Command
+      # Public: Initializes the command.
+
```

Comment:
```suggestion

      # Public: Initializes the command.
```

Reply:
Done

Reply:
Rubocop is not letting me add an indentation here.